bake.sh

```diff
@@ -63,6 +63,13 @@ elif [ "${FORMAT}" = "ext4" ] || [ "${FORMAT}" = "ext2" ]; then
   mkfs."${FORMAT}" -E root_owner=0:0 -d "${SYSEXTNAME}" "${SYSEXTNAME}".raw
   resize2fs -M "${SYSEXTNAME}".raw
 else
-  mksquashfs "${SYSEXTNAME}" "${SYSEXTNAME}".raw -all-root -noappend -xattrs-exclude '^btrfs.'
+  VER=$({ mksquashfs -version || true ; } | head -n1 | cut -d " " -f 3)
+  VERMAJ=$(echo "${VER}" | cut -d . -f 1)
+  VERMIN=$(echo "${VER}" | cut -d . -f 2)
+  ARG=(-all-root -noappend)
+  if [ "${VERMAJ}" -gt 4 ] && [ "${VERMIN}" -gt 6 ]; then
```

Comment:
The check should be `if [[ "${VERMAJ}" -gt 4 || ( "${VERMAJ}" -eq 4 && "${VERMIN}" -gt 5 ) ]]; then` ?

See:
https://github.com/plougher/squashfs-tools/blob/master/README-4.6.1#L37

Reply:
Yeah, your logic is correct.